src/UI/Component/Panel/Listing/Divider.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/* Copyright (c) 2017 Alex Killing <killing@leifos.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Panel\Listing;
+
+/**
+ * Listing divider
+ * @package ILIAS\UI\Component\Panel\Listing
+ */
+interface Divider extends \ILIAS\UI\Component\Component {
```

Comment:
I wouldn't add a general group for every kind of component, but a group for items (in this case). Would you still see the weak semantics in this case?

I fail to see how a your HTML heading example transfers to the structure we use for the UI framework. The divider in fact can never get hold on the contained items because to do that, I would need to go to the parent, get to know my location and then collect every item that comes after my divider up to the next divider. Which I can't.

Furthermore, a divider cannot possibly have a renderer, that e.g. wraps a div around the items to add an accordion-like expander or similar things. It can only be renderers as an insertion to the html and never wrap  around the contained items. For dropdowns this will prohibit the implementation of something like submenus (which is fine for me), for items this will prevent accordions and similar things (which to me is sad).

To me also this grouping functionality (for days in appointment listings, types in repositories, ...) is also not a heading or a divider. It really is a group (that might be renderer as a HTML heading in some cases). Also at least in the repository case we really talk about groups (or at least did so in the past).

Speaking on a very high level, a group is strictly more powerful than a divider. This is because a divider does know nothing about its location, while a group knows the items it spans.